### DISCUSSION TITLE
Explain why the event show "unknown/unsupported"

### DISCUSSION THREAD

**Discussion by Bouh** [4ian/GDevelop#4374] [Enhancement ideas (and their technical discussions)]
From the original idea: https://github.com/4ian/GDevelop/issues/3578#issuecomment-1030395533

## Needs

When an action or a condition from an extension is broken or if the event is present in the event sheet but the extension isn't present in the project there are no indications of why the event is broken.

![image](https://user-images.githubusercontent.com/1670670/194838531-bd6e95a3-e8d6-4f6f-bc98-07651ed0e829.png)

## Suggestion 

Add the source of the error in the event front of the "unknown/unsupported" message.

![image](https://user-images.githubusercontent.com/1670670/194838715-fba8eee3-8669-40e3-a012-07d6cd225d2f.png)

**Comment by LuniMoon** [comment]
New UI proposition: 
Giving the user visibility of system status (which is the specific error), and giving them the possibility to recover from it ("My extension name" opens the extension pop-up, so the user can install it directly).

We can even make the UX copy more specific saying "MyCoolAction from MyExtensionName missing. [Install it ](https://wiki.gdevelop.io/gdevelop5/extensions/read-pixels/reference)to continue" (the URL takes the user to the extension installation pop-up, **OR even better:** installs it automatically when the user clicks the underlined text).

<img width="696" alt="Screenshot 2022-10-10 at 12 20 19" src="https://user-images.githubusercontent.com/103995399/194845032-b9632673-8b20-4be6-b274-6d6b52d1d8f7.png">

_UI component uses Red50 and Body Small Accent for the error text_

**Comment by D8H** [comment]
It would be very useful to show the parameter expressions too, but we would have to guess the type.
